BookPlayer/Player/PlayerViewController.swift

```diff
@@ -177,7 +177,13 @@ class PlayerViewController: UIViewController, UIGestureRecognizerDelegate {
             items.append(self.sleepLabel)
         }
 
-        self.chaptersButton.isEnabled = self.currentBook.hasChapters
+        if self.currentBook.hasChapters {
```

Comment:
With this change, books that don't have chapters, have the chapters' button enabled, which shouldn't happen

Reply:
Ah my bad. Tested with one chapter book before pushing but missed item without any chapter. I have updated the button's default state to disabled in the story board so this works fine. LMK if this works.

Reply:
There's still one case not covered :/. If the user has autoplay in his library, or if it has in a playlist, a book with chapters, which is followed by a book with no chapters, when the player screen is reloaded for the next book (the one without chapters), the chapters button stays enabled

Reply:
Updated the crash on the corner case you mentioned. Tested the following scenarios

Autoplay order :
1. Book with chapters > 1  -> No chapter - > 1 chapter (Library/playlist) -> 2 Chapter
2. No chapter - >Book with chapters > 1 ->  1 chapter (Library/playlist) -> 2 Chapter 
3. Library No chapter item -> PLaylist(Book with chapters > 1) -> Library(1 chapter item)  -> Library(2 Chapter)

All of these have been tested with and without autoplay. Works as expected.